packages/remote-method-guesser/PKGBUILD

```
@@ -44,16 +44,19 @@ package() {
   install -dm 755 "$pkgdir/usr/share/$pkgname"
 
   install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" *.md
-  cp -a docs "$pkgdir/usr/share/doc/$pkgname/"
+  mv docs/README.md docs/README-docs.md
+  find docs -type f -exec install -m 644 -t "$pkgdir/usr/share/doc/$pkgname/" {} \;
 
-  cp -a target "$pkgdir/usr/share/$pkgname/"
+  install -Dm 644 target/rmg-5.1.0-jar-with-dependencies.jar "$pkgdir/usr/share/$pkgname/rmg-5.1.0-jar-with-dependencies.jar"
```

```suggestion
  install -Dm 644 rmg-?.?.?-jar-with-dependencies.jar "$pkgdir/usr/share/$pkgname/rmg.jar"
```

Maybe try something like that to not hardcode the version so that it is more maintainable.

---

`rmg-?.?.?-jar-with-dependencies.jar` needs `target`.